talentsearch/resources/js/tests/skills/SkillResults.test.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from "react";
+import { screen, fireEvent } from "@testing-library/react";
+import { fakeSkills } from "@common/fakeData";
+import { render } from "../testUtils";
+import SkillResults, { SkillBlock } from "../../components/skills/SkillResults";
+
+const skills = fakeSkills();
+
+function renderSkillResults(title: string) {
+  return render(
+    <SkillResults
+      addedSkills={[skills[0], skills[1]]}
+      skills={skills}
+      title={title}
+      handleAddSkill={async () => {}}
+      handleRemoveSkill={async () => {}}
+    />,
+  );
+}
+
+function renderSkillBlock(isAdded: boolean) {
+  return render(
+    <SkillBlock
+      isAdded={isAdded}
+      skill={skills[0]}
+      handleAddSkill={async () => {}}
+      handleRemoveSkill={async () => {}}
+    />,
+  );
+}
+
+describe("Skill Results Tests", () => {
+  test("should display the correct title", async () => {
+    const title = "Results";
+    renderSkillResults(title);
+    const element = screen.findByText(`${title} ${title.length}`);
```

Comment:
I don't think you meant to include `${title.length}` in the search. Which, frankly, given that the test passes, has me concerned we're doing this type of test wrong 😕

Reply:
Yeah, I tried changing it to `screen.findByTest("asblkdjf")` and it still passed 🤣 😭

Reply:
😆 I'll go over all the tests again. I think I got a little too happy seeing everything passing.

Reply:
Oh! It was because I was using `findByText()` instead of `getByText` 😆

Reply:
What does findByText do?

Reply:
They both look for the element, however findByText returns a Promise instead of the element. I think that might be why it always passed.

Reply:
Ah, the promise object was always truthy.